nixos/modules/services/web-apps/matomo.nix

```diff
@@ -36,7 +36,7 @@ in {
 
       package = mkOption {
         type = types.package;
-        description = "Matomo package to use";
+        description = "matomo package to use";
```

Comment:
Meh.. it's a descriptive text, not a variable. So it feels nicer to stay close to upstream name `Matomo` with the caps